### ISSUE TITLE
Number of people trained

### ISSUE THREAD

**Issue by mihai-sysbio** [elixir-europe-training/Training-Metrics-Database#37]
**Describe the solution you'd like**

As a Training Liaison, I want to see the number of people training, so that I can update this number on the [ELIXIR Training Platform page](https://elixir-europe.org/platforms/training).

**Describe alternatives you've considered**

The only option to do this now is to export the data for the first chart https://tmd.elixir-europe.org/demographic and add the answers. However, adds up to 12.2k+, whereas the "old" number is 38.7k+, so something is not adding up.

**Is your feature request related to a problem? Please describe.**

**Additional context**

**Comment by olisand** [comment]
Same as issue #62?

**Comment by mihai-sysbio** [comment]
Indeed, however the description of this one includes an indication of a mismatch.

**Comment by olisand** [comment]
So, we keep both but link them? I don't see how to do it without closing one at this point of my small GitHub knowledge. anybody feel free to do it :-)

**Comment by mihai-sysbio** [comment]
I've linked them by copying the comment - all good so proceeding with closing the issue. Please re-open if I missed anything.
